src/pages/viewer-app/viewer-app.tsx

```diff
@@ -216,7 +220,13 @@ export const ViewerApp = () => {
     const tilesetsData: TilesetMetadata[] = [];
 
     for (const layer of activeLayers) {
-      const params = parseTilesetUrlParams(layer.url, layer);
+      let params = { tilesetUrl: "" as string | File, token: "" };
+      if (typeof layer.url === "string") {
+        params = parseTilesetUrlParams(layer.url, layer);
+      } else {
+        params.tilesetUrl = layer.url;
+      }
```

Comment:
Why this part is different in `src/pages/debug-app/debug-app.tsx`  and `src/components/comparison/comparison-side/comparison-side.tsx`?